src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/serviceprovider/performance/PerformanceReportJobListener.kt

```diff
@@ -29,32 +30,51 @@ class PerformanceReportJobListener(
     val params = jobExecution.jobParameters.parameters
     val path = createTempDirectory().resolve("${params["user.id"]}_${params["timestamp"]}.csv")
 
-    logger.info("creating csv file for service provider performance report {}", StructuredArguments.kv("path", path))
+    logger.info("creating csv file for service provider performance report {}", kv("path", path))
 
     jobExecution.executionContext.put("output.file.path", path.toString())
   }
 
   override fun afterJob(jobExecution: JobExecution) {
-    if (jobExecution.status == BatchStatus.COMPLETED) {
-      val path = Path(jobExecution.executionContext.getString("output.file.path"))
-
-      s3Service.publishFileToS3(path, "reports/service-provider/performance/")
-
-      emailSender.sendEmail(
-        notifyTemplateId,
-        jobExecution.jobParameters.getString("user.email"),
-        mapOf(
-          "serviceProviderFirstName" to jobExecution.jobParameters.getString("user.firstName"),
-          "reportUrl" to UriComponentsBuilder.fromHttpUrl(interventionsUiBaseUrl)
-            .path(downloadLocation)
-            .buildAndExpand(path.fileName)
-            .toString()
-        )
-      )
+    val path = Path(jobExecution.executionContext.getString("output.file.path"))
+
+    when (jobExecution.status) {
+      BatchStatus.COMPLETED -> {
+        s3Service.publishFileToS3(path, "reports/service-provider/performance/")
 
-      // delete the file now the job is successfully completed
-      logger.info("deleting csv file for service provider report {}", StructuredArguments.kv("path", path))
-      File(path.toString()).delete()
+        emailSender.sendEmail(
+          successNotifyTemplateId,
+          jobExecution.jobParameters.getString("user.email"),
+          mapOf(
+            "serviceProviderFirstName" to jobExecution.jobParameters.getString("user.firstName"),
+            "reportUrl" to UriComponentsBuilder.fromHttpUrl(interventionsUiBaseUrl)
+              .path(downloadLocation)
+              .buildAndExpand(path.fileName)
+              .toString()
+          )
+        )
+      }
+      BatchStatus.FAILED -> {
```

Comment:
we get sentry notifications on issues in the job execution anyway, but should we also log an error here or something?